apps/examples/avs_test/avs_test_main.cpp

```diff
@@ -53,12 +53,14 @@
 #include <tinyara/config.h>
 #include <tinyara/init.h>
 #include <apps/platform/cxxinitialize.h>
-
 #include <fcntl.h>
 #include <wifi_manager/wifi_manager.h>
-
+#ifdef CONFIG_NETUTILS_NTPCLIENT
+#include <protocols/ntpclient.h>
+#endif
+#include <pthread.h>
+#include <time.h>
```

Comment:
i think it can move to line 59 in CONFIG_NETUTILS_NTPCLIENT block.

Reply:
Fixed, thanks.